xml/bp_troubleshooting_status.xml

```diff
@@ -866,6 +866,100 @@
       </listitem>
     </varlistentry>
   </variablelist>
+  <variablelist><title>HEALTH CHECKS</title>
+  <varlistentry>
+    <term><option>CEPHADM_PAUSED</option></term>
+    <listitem>
+      <para>
+        &cephadm; background work has been paused with <command>ceph orch pause</command>.
+        &cephadm; continues to perform passive monitoring activities (like
+        checking host and daemon status), but it will not make any changes
+        (like deploying or removing daemons).
```

Comment:
i'm splitting hair here, but the "likes" seem a bit colloquial to me
```suggestion
        &cephadm; continues to perform passive monitoring activities (for exeample,
        checking host and daemon status), but it will not make any changes
        (for example, deploying or removing daemons).
```